### PR TITLE
Added multiple stage deployment by tag

### PR DESCRIPTION
**Pull request by rodrigopavezi**
[stage-0]

@donnapep @stulees please review. cheers

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ reports/
 src/components/
 src/config/config.js
 src/*-e2e.html
+.idea
+widget-image.iml
```
Comment (stulees) [inline .gitignore]: @rodrigopavezi I use Webstorm also and I apply a global .gitignore so I can ensure specifics like the above don't have to be added to our repo git ignores as they're specific to your own IDE setup. Could you do the same?

Comment (rodrigopavezi) [inline .gitignore]: @stulees good to know. I have added to the globals. cheers

**Comment by rodrigopavezi** [comment]
@stulees @donnapep I made the changes, please review. cheers

**Comment by donnapep** [comment]
:+1:

**Comment by stulees** [comment]
@rodrigopavezi :+1:

**Pull request merged**